server/routes/optout/optOut.ts

```diff
@@ -0,0 +1,69 @@
+import { Request, Response } from 'express'
+import { path } from 'static-path'
+import { CaseAdminCaseloadService } from '../../services'
+import { convertToTitleCase } from '../../utils/utils'
+import OptOutReasonType from '../../enumeration/optOutReasonType'
+import { FieldValidationError } from '../../@types/FieldValidationError'
+import OptOutService from '../../services/optOutService'
+
+export default class OptOutRoutes {
+  constructor(
+    private readonly caseAdminCaseloadService: CaseAdminCaseloadService,
+    private readonly optOutService: OptOutService,
+  ) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const assessment = await this.caseAdminCaseloadService.getAssessmentSummary(
+      req?.middleware?.clientToken,
+      req.params.prisonNumber,
+    )
+
+    res.render('pages/optOut/optOut', {
+      assessment: {
+        ...assessment,
+        name: convertToTitleCase(`${assessment.forename} ${assessment.surname}`.trim()),
+      },
+    })
+  }
+
+  POST = async (req: Request, res: Response): Promise<void> => {
+    const { optOutReason, otherReason } = req.body
+
+    const assessment = await this.caseAdminCaseloadService.getAssessmentSummary(
+      req?.middleware?.clientToken,
+      req.params.prisonNumber,
+    )
+    const name = convertToTitleCase(`${assessment.forename} ${assessment.surname}`.trim())
+    const validationErrors = this.validateForm(optOutReason, otherReason)
```

Comment:
just gone for manual validation for this as it seemed less work doing this than setting up a validator. I can look at adding a validation library after if we're going to need to more complicated forms.

Reply:
I think post and redirect with flash state would be nicer, but this is alright for now